docs/index.md

```diff
@@ -1,14 +1,27 @@
-# COB Ontology Documentation
+# The Core Ontology for Biology and Biomedicine
 
 [//]: # "This file is meant to be edited by the ontology maintainer."
 
-This project is an attempt to bring together key terms from a wide range of [Open Biological and Biomedical Ontology (OBO)](http://obofoundry.org) projects into a single, small ontology. The goal is to improve interoperabilty and reuse across the OBO community through better coordination of key terms. Our plan is to keep this ontology small, but ensure that one or more COB terms can be used as the root of any given OBO library ontology.
+\[ [Browse COB on Ontobee](https://ontobee.org/ontology/COB) \] \[ [View COB on GitHub](https://github.com/OBOFoundry/COB/) \]
 
-## Editors Guide
+The Core Ontology for Biology and Biomedicine (COB) aims to bring together key terms from a wide range of [Open Biological and Biomedical Ontologies (OBO)](http://obofoundry.org) Foundry projects into a single, small ontology. COB anchors these terms in the Basic Formal Ontology (BFO) which ensures that COB based ontologies are compatible with BFO, but COB elects to omit some of the complexities of BFO that have proven confusing to users in the biomedical domain. Our goal is to improve interoperability and term reuse across the OBO community through better coordination of widely-used terms. We intend to keep this ontology small while ensuring that one or more COB terms can be used as the root of any given OBO Foundry ontology.
```

Comment:
drop "small" by "top-level"?

Reply:
I like 'small' here. Top level is now already spelled out above.

Reply:
OK!